buster/docparser.py

```diff
@@ -19,22 +20,22 @@
 supported_docs = {
     "mila": {
         "base_url": "https://docs.mila.quebec/",
-        "filename": "documents_mila.tar.gz",
+        "filename": "documents_mila.csv",
         "parser": SphinxParser,
     },
     "orion": {
         "base_url": "https://orion.readthedocs.io/en/stable/",
-        "filename": "documents_orion.tar.gz",
+        "filename": "documents_orion.csv",
         "parser": SphinxParser,
     },
     "pytorch": {
         "base_url": "https://pytorch.org/docs/stable/",
-        "filename": "documents_pytorch.tar.gz",
+        "filename": "documents_pytorch.csv",
         "parser": SphinxParser,
     },
     "huggingface": {
         "base_url": "https://huggingface.co/docs/transformers/",
-        "filename": "documents_huggingface.tar.gz",
```

Comment:
why are we switching back to csv format? Wasn't the point that .csv was very inefficient + took much more memory on disk?

Reply:
csv are human readable. Otherwise might as well just use the db. Probably I should just remove the filename field

Reply:
it's convenient to have the `tar.gz` format for deployment though

Reply:
And you can still use it. It's just that the filename refers to the step before the embedding currently, and csv are better for that. It's a bit confusing and should be reworked